### ISSUE TITLE
Ch09 movieclassifier - No module named 'sklearn.linear_model.stochastic_gradient'

### ISSUE THREAD

**Issue by myst10py** [rasbt/python-machine-learning-book-3rd-edition#161]
I believe the sklearn lib has updated it's naming convention of the stochastic gradient linear model to something else, which is hard coded in the classifier.pkl file of the movieclassifier directory.
